src/components/Editor/DebugLine.js

```diff
@@ -130,6 +135,9 @@ export class DebugLine extends Component<Props, State> {
 
 export default connect(state => ({
   selectedLocation: getSelectedLocation(state),
+
```

Comment:
I didn't get it. do we want to refactor getSelectedLocation to refer to original or sourcedMap location or change the getSelectedFrame to return location?